docs/structured-streaming-kafka-integration.md

```diff
@@ -607,5 +609,9 @@ and its dependencies can be directly added to `spark-submit` using `--packages`,
 
     ./bin/spark-submit --packages org.apache.spark:spark-sql-kafka-0-10_{{site.SCALA_BINARY_VERSION}}:{{site.SPARK_VERSION_SHORT}} ...
 
+For experimenting on `spark-shell`, you can also use `--packages` to add `spark-sql-kafka-0-10_{{site.SCALA_BINARY_VERSION}}` and its dependencies directly,
```

Comment:
oh. the `you can also use` is relatively to `spark-submit`. Above this, `spark-submit` uses `--packages` to add library, so here it says you can also use this option to add library.